kernel/src/hil/qdec.rs

```diff
@@ -0,0 +1,31 @@
+//! Interface for a Qdec compatible chip
+//!
+//! This trait provides a stanfard interface for chips with a
+//! quadrature decoder. Any interface functions that
+//! a chip cannot implement can be ignored by the chip capsule
```

Comment:
I would not say that any functions a chip cannot implement can be ignored, as some of them not being implemented is likely to break the userspace driver. Instead, I would just include a note that this interface is still experimental and may need to be updated once it is implemented on additional chips.

Reply:
done!